webpack.dev.js

```diff
@@ -15,6 +15,9 @@ module.exports = merge(common, {
               sourceMap: true,
             },
           },
+          {
+            loader: 'postcss-loader',
```

Comment:
This needs to be in `webpack.common.js` I think— as-configured we'll lose prefixes in prod. Part of that will be verifying that `wepack-merge` is configured to handle everything correctly. If that's too much, you can also just duplicate this across `webpack.dev.js` and `webpack.prod.js` and we can resolve it later.